pyvo/utils/http.py

```diff
@@ -6,14 +6,14 @@
 from ..version import version
 
 
-def requests_session(useragent=None):
+def requests_session(useragent=None, cookie=None):
     if not useragent:
         useragent = 'python-pyvo/{}'.format(version)
 
     session = requests.Session()
     session.headers['User-Agent'] = useragent
```

Comment:
I think this file slipped in your commit by mistake (plus, on line 18 it's failing the flake8 check because it likes that extra blank line there).  I'm going to be working on the auth very soon though, it's next on my plate.

Reply:
argh, yes, this is the danger of `git commit -a`

Reply:
`git commit -a` should be 🔥-ed as an option...